src/Cli/ConfigGenerator.cs

```diff
@@ -934,7 +936,7 @@ public static bool TryStartEngineWithOptions(StartOptions options, RuntimeConfig
 
             /// This will add arguments to start the runtime engine with the config file.
             List<string> args = new()
-            { "--ConfigFileName", runtimeConfigFile };
+            { "--" + nameof(RuntimeConfigLoader.CONFIGFILE_NAME), runtimeConfigFile };
```

Comment:
This doesn't seem right, should be either `-c` `--config` 
![image](https://github.com/Azure/data-api-builder/assets/6414189/be03a9d2-aca4-40e2-b0ef-5da8241ea0bb)

Reply:
Yes, and I thought I fixed that in the branch for #1402, so I'll check this branch still works and is up to date.

Reply:
Checked against main, this I think is wrong in `main` too... but shouldn't tests fail?

Reply:
It is an incorrect value but it's not that it should be `config` it should be `ConfigFileName` to match https://github.com/Azure/data-api-builder/blob/2e8d1bb8b7d83bbe2a90472c2ccdc8acb87fa5ec/src/Service/Startup.cs#L61

This change is the result of a mistaken merge with the branch #1402, so I'll revert.

Reply:
> This doesn't seem right, should be either `-c` `--config` ![image](https://user-images.githubusercontent.com/6414189/250363089-be03a9d2-aca4-40e2-b0ef-5da8241ea0bb.png)

The args for starting cli are `-c`, `--config` using `dab start`. 
However, the args for running the engine directly used `--ConfigFileName` so we should continue to support both to avoid regression

Reply:
We do support both, but in the case where this line of code is called it's only launching the service using information provided from the CLI.